Makefile

```diff
@@ -292,10 +264,10 @@ snapcraft-local.yaml: snapcraft-local.yaml.in
 # unless there is a reason not to.
 # https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
 .PHONY: build install install_debug dist clean distclean \
-check_tools check_dev_tools get_vendor_deps draw_deps test test_cli test_unit \
-test_cover test_lint benchmark devdoc_init devdoc devdoc_save devdoc_update \
+draw_deps test test_cli test_unit \
```

Comment:
Should `tools` be in this list?

Reply:
it's in `scripts/Makefile`'s `.PHONY`. That should be enough